### ISSUE TITLE
Replace kleur with styleText

### ISSUE THREAD

**Issue by benmccann** [sveltejs/kit#12396]
### Describe the problem

Can use node built-in in favor of extra dependency 

### Describe the proposed solution

Requires Node 20

https://stackoverflow.com/a/78604048

### Importance

nice to have

**Comment by sidharthv96** [comment]
Is there a branch, similar to `version-2` for https://github.com/sveltejs/kit/pull/11172, that can be used to upgrade node from 18.13 to 20.13?

**Comment by benmccann** [comment]
No. We need to wait until we're ready to do the next major release. There's no timeline on that currently.
